app/views/comments/_comment_form.html.haml

```diff
@@ -7,5 +7,3 @@
   %p
     = link_to "How to comment", @application.comment_url
     on this application (from #{@application.authority.full_name}).
-    - if @application.on_notice_to
```

Comment:
Oh, one other thing :) Those two commits should be one commit moving the notice, rather than delete and insert. That way the whole intention is captured in the single commit we can look back on.